Source/ACE/Entity/Landblock.cs

```diff
@@ -431,7 +431,7 @@ private void HandleGameAction(QueuedGameAction action, Player player)
                                         // validate within use range
                                         float radiusSquared = obj.GameData.UseRadius * obj.GameData.UseRadius;
 
-                                        if (player.Location.SquaredDistanceTo(obj.Location) <= radiusSquared)
+                                        if (player.Location.SquaredDistanceTo(obj.Location) >= radiusSquared)

```

Comment:
Think about what this now means... If the player is FURTHER away than the object radius...

The "<=" was correct, it just wasn't taking the size of the lifestone into account.  If you add a fudge factor of 1.5 to the useRadius, this should correct the issue.  Here's an example:

float modelSize = 1.5f;
float radiusSquared = (modelSize + obj.GameData.UseRadius) * (modelSize + obj.GameData.UseRadius);